shell/shared/shell.js

```diff
@@ -935,6 +1008,16 @@ Router.map(function () {
     }
   });
 
+  this.route("referrals", {
+    path: "/referrals",
+
+    waitOn: function() {
+      return [Meteor.subscribe("myReferrals"),
```

Comment:
Yeah, a publish can't return >1 cursor into the same collection, which is why I made this multiple subscriptions.

I'll work on making it a pseudo-collection. I've never made a pseudo-collection before!